lightwood/api/json_ai.py

```diff
@@ -1030,9 +1030,7 @@ def code_from_json_ai(json_ai: JsonAI) -> str:
 if self.problem_definition.fit_on_all:
 
     log.info("Adjustment on validation requested.")
-    update_data = {{"new": enc_train_test["test"], "old": ConcatedEncodedDs([enc_train_test["train"], enc_train_test["dev"]])}}  # noqa
-
-    self.adjust(update_data)
+    self.adjust(enc_train_test["test"], ConcatedEncodedDs([enc_train_test["train"], enc_train_test["dev"]]))
```

Comment:
we need to revamp concatenatedencodedds

also the data structure is misspelled : /

Reply:
self consistently fine; it's called this in the script that defines the class